modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -621,6 +679,7 @@ protected void readdForcedExclusions() throws Exception {
 	private Object _outputDir;
 	private Object _pomArtifactId;
 	private Object _pomGroupId;
+	private final Map<String, String> _pomRepositories = new HashMap<>();
```

Comment:
Instead of a `HashMap`, I would use a `LinkedHashMap` (google to find the difference between the two). I think the order matters for repositories...